src/main/java/com/hisabkitab/entity/Debt.java

```diff
@@ -0,0 +1,80 @@
+package com.hisabkitab.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "debt")
+public class Debt {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(name = "group_id", referencedColumnName = "id")
+    Group groupId;
+    @ManyToOne
+    @JoinColumn(name = "owed_by", referencedColumnName = "id")
+    User owedBy;
+    @ManyToOne
+    @JoinColumn(name = "owed_to", referencedColumnName = "id")
+    User owedTo;
+    private Double amount;
+
+    public Debt(Long id, Group groupId, User owedBy, User owedTo, Double amount) {
```

Comment:
We can use annotations @Data, @RequiredArgsConstructor, @NoArgsConstructor to generate these methods and constructors, instead of creating them manually